### ISSUE TITLE
DOC on s3 retention for logs

### ISSUE THREAD

**Issue by nityanandagohain** [SigNoz/signoz#1553]
We don't have any docs today for setting s3 retention for logs - may be we can update this once our s3 perf benchmarks for logs is done - https://www.reddit.com/r/selfhosted/comments/x856uk/comment/injeu18

**Comment by nityanandagohain** [comment]
Duplicate of [Coldstorage query performance of logs#643](https://github.com/SigNoz/engineering-pod/issues/643)
